\documentclass[11pt]{article}
\usepackage{amsmath,amssymb,amsthm}
\usepackage{graphicx}
\usepackage[margin=1in]{geometry}
\usepackage[table]{xcolor}
\usepackage{csvsimple}
\usepackage{enumitem}
\usepackage{float}
\usepackage{caption}
\usepackage{subcaption}
\restylefloat{table}
\usepackage{fancyhdr}
\setlength{\parindent}{0pt}
\setlength{\parskip}{5pt plus 1pt}
\setlength{\headheight}{13.6pt}
\newcommand\question[2]{\vspace{.25in}\hrule\textbf{#1: #2}\vspace{.5em}\hrule\vspace{.10in}}
\renewcommand\part[1]{\vspace{.10in}\textbf{(#1)}}
\newcommand\algorithm{\vspace{.10in}\textbf{Algorithm: }}
\newcommand\correctness{\vspace{.10in}\textbf{Correctness: }}
\newcommand\runtime{\vspace{.10in}\textbf{Running time: }}
\pagestyle{fancyplain}
\lhead{\textbf{\NAME\ (\ANDREWID)}}
\rhead{Page \thepage}
\begin{document}


%Section A==============Change the values below to match your information==================
\newcommand\NAME{Turaga}  % your name
\newcommand\ANDREWID{nturaga1@jhmi.edu}     % your andrew id
%\newcommand\HWNUM{1}              % the homework number

\title{Shyam Biswal : MeDIP on 2.1M Nimblegen Array}
\author{Nitesh Turaga}
\maketitle


%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Data Description}
%%%%%%%%%%%%%%%%%%%%%%%%%%
Data from Shyam Biswal, used to compare the differential methylation regions of H460 Knock-in cell line and H460 parent cell line.

The data was generated from two color Nimblegen 2.1M oligonucleotide microarrays. The raw Nimblegen data was in the form of {\tt .tif} images and the corresponding array design files were given for building the annotation package {\tt 100929\_HG19\_Deluxe\_Prom\_Meth\_HX1} by Nimblegen. 

There were a total of 6 samples, 3 for each cell line(H460 knock-in and H460 parent). The 2.1M Platform refers to the high-density Roche NimbleGen 2.1 million-feature arrays, i.e, 2.1M probes on each array 

Array images were processed with DEVA-v1.2 (Nimblegen software for automated feature extraction and data analysis). The TIF files were processed and converted to {\tt .xys} files for analysis. {\tt .xys} files report the, {\bf X} - coordinate of the feature on the image, {\bf Y}-coordinate of the feature on the image, and the {\bf S}ignal - the flourescence intensity of the pixels that make the feature.


The TIF files were also processed with DEVA using the DNA methylation work flow to identify peaks and generate a result for each sample.

%%%%%%%%%%%%%%%%%%%%%%%%%% 
\section*{Preliminary Assessments}
%%%%%%%%%%%%%%%%%%%%%%%%%%


General QC-analyses brought to light that the signal intensities and local enrichment at methylated sites were not really large. Initial array quality assessment was done using the Bioconductor {\tt charm-package} on the {\tt .xys} files. The data quality can be assessed by looking at the Enriched channel in the MeDIP array, where we expect every probe to have a signal. Since, the enriched channel contains the only methylated DNA, a successful hybridization would indicate a strong signal. The array signal is calculated as the average percentile rank of the signal probes among the background probes. The score ranges between 0 to 100, where 100 indicates the ideal scenario or perfect hybridization. This quality score is calculated before any kind of normalization is done on the arrays.

\begin{table}[H]
    \begin{tabular}{|l|l|l|}
    \hline
    Sample ID                        & Status       & Quality Score     \\ \hline
    542586A01\_Slot9\_2012-07-24\_H460 & H460\_parent & 59.6804601609024  \\ \hline
    542654A01\_Slot9\_2012-07-25\_H460 & H460\_knockin & 74.9611413105766  \\ \hline
    542671A01\_Slot6\_2012-07-24\_H460 & H460\_knockin & 71.2294097946914  \\ \hline
    542707A01\_Slot12\_2012-07-25\_H460 & H460\_knockin & 76.519565668383   \\ \hline
    542737A01\_Slot13\_2012-07-24\_H460 & H460\_parent & 64.4412397468657  \\ \hline
    542857A01\_Slot18\_2012-07-24\_H460 & H460\_parent & 69.8646821319654  \\ \hline
    \end{tabular}
    \caption{Array quality scores}
\end{table}

\includegraphics[scale=0.5]{qcImage1.png}

\includegraphics[scale=0.5]{qcImage2.png}


As we can in Table 1, the quality score of the H460 Parent arrays are pretty low, which indicates hybridization problems.

%%%%%%%%%%%%%%%%%%%%%%%%%% 
\section*{The CHARM Algorithm}
%%%%%%%%%%%%%%%%%%%%%%%%%%

Array quality scores were generated with the  Bioconductor {\tt charm - package}, and data quality was checked using the $Enriched$ channel. Four of the six arrays show a large standard deviation in the signal strenth and seem to have a problem in hybridization.

To estimate the DNA methylation values, the background signal is removed before computing the log-ratios. A within sample normalization method - {\bf loess} is used, and then a between array normalization - quantile  is used. The control probes which are the non-CpG probes are excluded. This provides us a probe-level estimate of DNA methylation from the 2.1M oligonucliotide microarray.

After estimating the DNA methylation values in terms of percentage methylation, we use the regression based DMR-finding approach after correcting for batch effects, which is provided by {\tt charm}. This fails to find DMR’s in the samples. Also, while not taking into account batch effects (surrogate variables), {\tt charm} does not find any DMR's. 



%%%%%%%%%%%%%%%%%%%%%%%%%% 
\section*{The Bumphunter Algorithm}
%%%%%%%%%%%%%%%%%%%%%%%%%%


Bumphunter is used to estimate regions for which the genomic profile deviates from a baseline value(cut off). It is implemented to detect differntially methylated genomic regions between two populations. It is also a regression based approach.

It reports the result, with a table of candidate differentially methylated regions and the corresponding annotation for wach region.

{\bf Analysis using Bumphunter}


\begin{enumerate}

\item Run  1

Initially ran {\tt bumphunter} on the data, with cutoff value 1.0, this failed to find any bumps. The sensitivity of the cutoff was not enough to catch any DMRs in the sample set. Bumphunter is better used for large sample sets, for better performance. 


\item Run  2

Bumphunter, with the inbuilt argument to pickCutoff was run. The cutoff chosen by bumphunter was at 0.41, bumps found 28206. The number of DMRs/bumps of length greater than four is 0. This result is not useful for analysis as the length of the DMRs are too small.

%bumps.rda file attached. 

The distribution of the length of DMRs found, is shown below. Most of them are single CpG sites of length 1. This does not work for a significant analysis.

\end{enumerate}

\begin{table}[H]
	\begin{tabular}{|l|l|l|l|}
	\hline
	Length(L) &   1     & 2   & 3 \\ \hline
    Number of DMRs & 27981 & 222 & 3 \\ \hline
	\end{tabular}
	\caption{Distribution of Bumps}
\end{table}

\includegraphics[scale=0.4]{bumpsDistribution.png}

%\begin{verbatim}
%Argument description for bumphunter,
%
%cutoff: 
%
%A numeric value. Values of the estimate of the genomic profile above the cutoff or
%below the negative of the cutoff will be used as candidate regions. It is possible to
%give two separate values (upper and lower bounds). If one value is given, the lower
%bound is minus the value.
%
%pickCutoff: 
%
%Should bumphunter attempt to pick a cutoff using the permutation
%distribution?
%
%\end{verbatim}

\vspace{0.2in}


%%%%%%%%%%%%%%%%%%%%%%%%%% 
\section*{The Nimblegen Algorithm}
%%%%%%%%%%%%%%%%%%%%%%%%%%

The standard Nimblegen algorithms were used to compute the normalized data and identify peaks of enrichment, coinciding with methylated regions. Next the data was transformed into a more usable format, i.e. the peaks near known transcription start sites (TSSs) were identiﬁed, according to 2 different cutoffs for the maximal distance between a peak and a TSS:
\begin{itemize}
	\item  -500 to +500, the default cutoff
	\item  -5000 to +1500, a custom cutoff
\end{itemize}

Peaks were identified for each sample using both of these cut offs. 

{\bf Analysis using Peaks files from DEVA}

For each cell line, the genes which were common in each sample were identified, i.e, gene names which intersect. All the genes which were identified jointly in both cell lines were removed, making each gene set unique and exclusive to the specific cell line. Both sets of genes, were ordered by distance from the Transcription start site. 944 genes were identified in H460 knock-in cell line and 349 in H460 parent. (Table 1)

%Sequential intersections, based on the number of peaks identified in each sample, was done in decreasing order for each experimental status. This step results in all the genes within each sample which are intersecting.

%We find the common genes between “H460\_knock-in” and “H460\_parent", and remove these genes from the intersected lists. We then order them by distance from the transcription start site(TSS).


\begin{table}[H]
\centering
	\begin{tabular}{|l|l|l|}
	\hline
		Cell line & Feature Track & Number of features \\ \hline
		H460 Knock in & transcription start site & 944 \\ \hline
		H460 Parent & transcription start site & 349 \\ \hline
	\end{tabular}
	\caption{Number of features in each cell line}
\end{table}

{\tt NOTE:  Files are attached as “knock-in\_genes\_distFromTSS.csv” and “parent\_genes\_distFromTSS.csv”. The regions are ordered in a decreasing order based on the distance from TSS.}


The only features left in the result are transcription start sites, other possible feature is CpG Island (column name  is  FEATURE TRACK). The distribution of the distance from the TSS is also shown for both cell lines. Column name is SHORTEST\_DISTANCE\_FROM\_FEATURE\_TO\_DATA\_POINT.


\begin{table}[H]
\centering
    \begin{tabular}{|l|l|l|l|l|l|l|}
    \hline
     Cell line & Min. & 1st Qu. & Median & Mean & 3rd Qu. &   Max.  \\ \hline
   H460 Knock in  & -4998.0 & -3363.0 & -1916.0 & -1956.0 & -569.8 & 995.0  \\ \hline
   H460 Parent & -4975 &  -3669 &  -2474 &  -2223 &   -718  &   997  \\ \hline
    \end{tabular}
    \caption{Summary of Feature distance from data point}
\end{table}


\begin{figure}[H]
\centering
\begin{subfigure}{.4\textwidth}
  %\centering
  \includegraphics[scale=0.3]{distribution_knocking.png}
  \caption{H460 knock-in distance from TSS}
  \label{fig:sub1}
\end{subfigure}
\quad
\begin{subfigure}{.4\textwidth}
  %\centering
  \includegraphics[scale=0.3]{distribution_parent.png}
  \caption{H460 parent distance from TSS}
  \label{fig:sub2}
\end{subfigure}

\caption{Distribution of Shortest distance from feature to data point}
\label{fig:test}
\end{figure}





%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Pathway Comparisons}
%%%%%%%%%%%%%%%%%%%%%%%%%%

Both the list of H460 Knock-in genes and H460 parent genes were compared with a list of selected pathways.

\begin{enumerate}[itemsep=-0.2mm]
\item Cell Cycle                                 
\item Complete Homeobox (HOX) Genes                      
\item Complete Human Inflammatory Response and Autoimmunity 
\item Complete Human Tumor Suppressor Genes              
\item Complete Stem Cell Transcription Factors            
\item Complete Stress and Toxicity                         
\item Cytokine Production                                
\item DNA Repair 
\item Human Epithelial to Mesenchymal Transition (EMT)    
\item Human Notch Signaling Pathway                      
\item Human T-Cell B-Cell Activation Methylation         
\item Human T Helper Cell Differentiation                
\item Human Tumor Suppressor Genes                        
\item Inflammatory Response and Autoimmunity               
\item Polycomb and Trithorax Complexes                     
\item Stem Cell Transcription Factors                    
\item TGF f BMP Signaling Pathway                          
\item Toll-Like Receptor Signaling Pathway               
\item WNT Signaling Pathway                              
\end{enumerate}

Very few pathway genes matched with the H460 knock-in genes and the H460 parent genes. 

The H460 knock-in genes have only 29 genes in common with the pathways(listed in Table 5). The following result from shows you the pathways and the genes along with gene description. The result, for the names of the genes is stored in "pathway\_genes\_knock.csv". 

\begin{table}[H]
	\resizebox{\columnwidth}{!}{%
    \begin{tabular}{|l|l|l|}
    \hline
    {\bf Gene name}   & {\bf Description}                                                                           & {\bf Pathway}                                  \\ \hline
    BRCA1  & breast cancer 1, early onset                                                          & Cell\_Cycle                              \\
    HOXB2  & homeobox B2                                                                           & Complete\_Homeobox\_(HOX)\_Genes         \\
    HOXB3  & homeobox B3                                                                           & Complete\_Homeobox\_(HOX)\_Genes         \\
    HOXB4  & homeobox B4                                                                           & Complete\_Homeobox\_(HOX)\_Genes         \\
    HOXB6  & homeobox B6                                                                           & Complete\_Homeobox\_(HOX)\_Genes         \\
    HOXB7  & homeobox B7                                                                           & Complete\_Homeobox\_(HOX)\_Genes         \\
    HOXB8  & homeobox B8                                                                           & Complete\_Homeobox\_(HOX)\_Genes         \\
    BRCA1  & breast cancer 1, early onset                                                          & Complete\_Human\_Tumor\_Suppressor\_Genes \\
    DIRAS3 & DIRAS family, GTP-binding RAS-like 3                                                  & Complete\_Human\_Tumor\_Suppressor\_Genes \\
    XRCC1  & X-ray repair complementing defective repair in Chinese hamster cells 1                & Complete\_Human\_Tumor\_Suppressor\_Genes \\
    GATA6  & GATA binding protein 6                                                                & Complete\_Stem\_Cell\_Transcription\_Factors \\
    HOXB3  & homeobox B3                                                                           & Complete\_Stem\_Cell\_Transcription\_Factors \\
    HOXB8  & homeobox B8                                                                           & Complete\_Stem\_Cell\_Transcription\_Factors \\
    SOX9   & SRY (sex determining region Y)-box 9                                                  & Complete\_Stem\_Cell\_Transcription\_Factors \\
    STAT3  & signal transducer and activator of transcription 3 (acute-phase response factor)      & Complete\_Stem\_Cell\_Transcription\_Factors \\
    ATF4   & activating transcription factor 4 (tax-responsive enhancer element B67)               & Complete\_Stress\_\&\_Toxicity           \\
    BRCA1  & breast cancer 1, early onset                                                          & Complete\_Stress\_\&\_Toxicity           \\
    ERCC1  & excision repair cross-complementing rodent repair deficiency, complementation group 1 & Complete\_Stress\_\&\_Toxicity           \\
    GPX7   & glutathione peroxidase 7                                                              & Complete\_Stress\_\&\_Toxicity           \\
    SMC1A  & structural maintenance ofchromosomes 1A                                               & Complete\_Stress\_\&\_Toxicity           \\
    XRCC1  & X-ray repair complementing defective repair in Chinese hamster cells 1                & Complete\_Stress\_\&\_Toxicity           \\
    ELANE  & elastase, neutrophil expressed                                                        & Cytokine\_Production                     \\
    BRCA1  & breast cancer 1, early onset                                                          & DNA\_Repair                              \\
    XRCC1  & X-ray repair complementing defective repair in Chinese hamster cells 1                & DNA\_Repair                              \\
    PSENEN & presenilin enhancer 2 homolog (C. elegans)                                            & Human\_Notch\_Signaling\_Pathway         \\
    BRCA1  & breast cancer 1, early onset                                                          & Human\_Tumor\_Suppressor\_Genes          \\
    CBX2   & chromobox homolog 2 (Pc class homolog, Drosophila)                                    & Polycomb\_\&\_Trithorax\_Complexes       \\
    STAT3  & signal transducer and activator of transcription 3 (acute-phase response factor)      & Stem\_Cell\_Transcription\_Factors       \\
    IRAK1  & interleukin-1 receptor-associated kinase 1                                            & Toll-Like\_Receptor\_Signaling\_Pathway  \\ \hline
    \end{tabular}}
     \caption {H460 Knock-in gene with corresponding pathways }
\end{table}


The H460 parent genes have only 2 genes in two pathways(Table 6) , and stored in "pathways\_genes\_parent.csv".


\begin{table}[H]
    \begin{tabular}{|l|l|l|}
    \hline
    {\bf Gene name} & {\bf Description}       & {\bf Pathway}                                  \\ \hline
    AR   & androgen receptor & Complete\_Stem\_Cell\_Transcription\_Factors \\ 
    AR   & androgen receptor & Stem\_Cell\_Transcription\_Factors       \\ \hline
    \end{tabular}
    \caption {H460 parent gene with corresponding pathways }
\end{table}


%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Plots and Results}
%%%%%%%%%%%%%%%%%%%%%%%%%%

Circos plot was made using RCircos, the inner track refers to the H460 parent genes and the outer track refers to the H460 knock-in genes.

{\bf NOTE:} Everything on the inside of the chromosome ideogram is the inner track. Everything on the outside of the ideogram is the outer track. Due to scaling issues, few genes have been omitted. Please refer the data files for more accurate results.

As it can be seen from the circos plot, very few chromosomes contribute to the methylated regions, including the sex chromosomes. The plot next to the gene names correspond to the peak values. It is easy to infer from this plot where there is a high frequency of methylated regions.

The pathways and the corresponding gene in that particular pathway are also plotted in the Circos plot in the inner and the outer tracks respectively.

{\bf REFER: BiswalCircosMain.png}


\includegraphics[scale = 0.25]{BiswalCircosMain.png}



%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{R-packages Used}
%%%%%%%%%%%%%%%%%%%%%%%%%%

List of R-packages used for analysis:
\begin{enumerate}

\item Charm

\item Bioconductor

\item BiocGenerics

\item plyr

\item RCircos

\end{enumerate}

%Extracting the percentage methylation values from the raw data was done using {\tt charm::methp}, where the default arguments were used to normalize. The normalization methods included were spatial normalization(to correct for spatial artifacts), background subtraction(to estimate and remove the background signal before computing the log-ratios), loess within sample normalization and quantile between sample normalization.
%
%These percentage methylation values are on a logit scale.
%
%A regression based DMR-finding after correcting for batch effects pipeline was used in this analysis. Removing batch effects and using surrogate variables in finding DMRs ({\tt refer sva-package}) have been shown to reduce dependence on unknown noise in the data set.



%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{References}  
%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{enumerate}


	\item  Aryee MJ et al., Accurate genome-scale percentage DNA methylation estimates
	  from microarray data, Biostatistics (2011) 12(2): 197-210 	

	\item  Seth Falcon, Benilton Carvalho with contributions by Vince Carey, Matt
	  Settles and Kristof de Beuf. pdInfoBuilder: Platform Design Information
	  Package Builder. R package version 1.24.0.
	
	\item   Rafael A. Irizarry, Martin Aryee, Hector Corrada Bravo, Kasper D. Hansen and	Harris A. Jaffee (). bumphunter: Bump Hunter. R package version 1.0.0.
		
	\item RCircos: an R package for Circos 2D track plots

\end{enumerate}

 
 
 
 
 \end{document}


